libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -112,3 +114,38 @@ void AP_GPS_Backend::fill_3d_velocity(void)
     state.velocity.z = 0;
     state.have_vertical_velocity = false;
 }
+
+
+void AP_GPS_Backend::_detection_message(char *buffer, const uint8_t buflen) const
+{
+    const uint8_t instance = state.instance;
+    const struct AP_GPS::detect_state dstate = gps.detect_state[instance];
+
+    if (dstate.detect_started_ms > 0) {
```

Comment:
I have none of these to physically test with.

I also can't find @WickedShell 's comments...

@WickedShell Is there some other way of filling in baud here?  Or should we special-case these guys?